examples/operator/README.md

```diff
@@ -2,7 +2,7 @@
 
 ```
 # Start minikube
-minikube start --cpus=8 --memory=11000 --disk-size=50g --kubernetes-version=v1.19.5
+minikube start --cpus=8 --memory=11000 --disk-size=50g --kubernetes-version=v1.26.3
```

Comment:
The latest supported version of the operator is `1.25.8` (https://github.com/planetscale/vitess-operator#compatibility), let's change it to that.

```suggestion
minikube start --cpus=8 --memory=11000 --disk-size=50g --kubernetes-version=v1.25.8
```